content/components/tokens.mdx

```diff
@@ -23,99 +36,167 @@ import {LabelGroup, Label, Box} from '@primer/components'
 </LabelGroup>
 
 ## Overview
+
 Tokens almost always appear in a group. A group of tokens may be read-only, or they may be used as a way to input data.
 
 A read-only group of tokens organizes information in a way that helps users quickly scan and understand the view they're on. For example, a list of topics associated with a repository.
 
 Tokens may be used for data input to quickly add and remove items to a list of related attributes. For example, defining a list of GitHub users who can access a private repo.
 
 ## Anatomy
-<Box display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 2}}>
-    <img
-        width="456"
-        alt="token anatomy diagram"
-        src="https://user-images.githubusercontent.com/2313998/137778682-ee35f8e0-72de-4a24-b1bd-e4ea02e2ac49.png"
-    />
-    <Box as="ul" mt="0">
-        <Box as="li" mt="0"><strong>Leading visual</strong> (optional): Provides additional context (e.g.: user avatar)</Box>
-        <Box as="li"><strong>Label</strong> (required): Concise text label</Box>
-        <Box as="li"><strong>Remove button</strong> (optional): Removes the token from an editable group of tokens</Box>
+
+<Box
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 2}}
+>
+  <img
+    width="456"
+    alt="token anatomy diagram"
```

Comment:
I think we either need to describe the diagram (if we feel the user would benefit from that context on top of the following description of the fields) or we need to make this a presentation image / give it an empty alt text.